core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-search-list/dot-category-field-search-list.component.scss

```diff
@@ -1,11 +1,31 @@
+@use "variables" as *;
+
 .category-field__search-list {
     display: flex;
     flex-direction: column;
     height: 100%;
     overflow: hidden;
 }
 
-:host ::ng-deep .dotTable.p-datatable > .p-datatable-wrapper {
-    border-radius: 0;
-    border: none;
+:host {
+    ::ng-deep {
+        .dotTable.p-datatable > .p-datatable-wrapper {
+            border-radius: 0;
+            border: none;
+        }
+    }
+
+    &.category-field__search-list--empty ::ng-deep {
+        p-table,
+        .dotTable.p-datatable,
+        .p-datatable-wrapper,
+        table {
+            height: 100%;
+
+            tr:not(.p-highlight):hover {
+                background: $white;
+                cursor: auto;
+            }
+        }
+    }
```

Comment:
Should we create an extra CSS class for this kind of style? We should double-check with @melissarojas-dotcms

Reply:
this is just for using the empty custom template of a table (primeNg). I need the table 100% of the container for the center of the design done by @melissarojas-dotcms.